src/overview.md

```diff
@@ -1,21 +1,48 @@
 
 ![](img/PublishFeaturesToAugurk-AddTask.png)
 
-Once added, a couple of parameters need to be filled in:
+## 4. Configure an Augurk connection
+After adding the task to the build definition you should configure a connection to Augurk. This can be done by using the + button next to the Augurk Instance option in the build definition editor. This will open up a dialog where you can enter the URL on which your Augurk instance is available and any credentials you need to supply in order to access Augurk.
+
+Once the connection has been established make sure it is selected as the Augurk instance to which features will be published.
+
+## 5. Setup publishing
+Finally, you'll need to configure a couple of parameters that are necessary for successfully publishing features.
+
+![](img/PublishFeaturesToAugurk-BuildTask.png)
+
+* **Feature(s)** - The set of feature files that need to be published. By default this is any .feature file in any directory.
+* **Augurk Instance** - Connection (created above) to the Augurk instance that you want to publish the feature files to.
+* **Product Name** - Name of the product under which the features files are to be published. You can select an existing product in Augurk, or enter the name of a new product.
+* **Product Description** - Optionally specify the path to a Markdown file here that is published as the description for the previously selected product in Augurk.
+* **Use folder structure** - Check this option if you want to use the existing folder structure as the names of groups within your Augurk.
+* **Group Name** - If the **Use folder structure** option isn't checked, enter the name of the group that you want to publish these feature files under in Augurk.
+* **Version** - Version number of the feature files being published. Ideally a build variable is used here so that the version matches the version of the build. Can also be left blank if you only want to see the latest published version of features in Augurk.
+* **Embed images** - When checked images that are referenced in feature files (using Markdown syntax) are embedded and published together to Augurk so that they are visible there.
+
+## Advanced settings
+To further customize how feature files are published to Augurk you can use the following advanced settings:
 
-![](img/PublishFeaturesToAugurk-BuildTask.png)
\ No newline at end of file
+* **Use integrated security** - When checked the command line tool will use integrated security when communicating with Augurk. This can be useful in on-premises scenario's where you want to limit who can publish feature files to Augurk.
+* **Language** - A .NET compatible culture name that is used to parse the feature files. Defaults to *en-US*. This is useful if you have your feature files written in another language but don't want to specify this language in every feature file.
```

Comment:
The new Gherkin parser we've been using for a while now does this yes. If there is a #language:<code> (I believe the format is) comment at the top of the file it will pick that up. But we had this option previously and I still think its useful to have if you don't want to put that comment in every feature file, but want a different default than English.